backend/officehoursqueue/settings/proddev.py

```diff
@@ -0,0 +1,17 @@
+from officehoursqueue.settings.production import *
```

Comment:
Whats the reason for adding this new `proddev.py`

Reply:
I don't believe it's possible to override Django settings with just environment variables without changing settings files.